ruma-client-api/src/r0/push/get_pushrule.rs

```diff
@@ -31,7 +32,7 @@ ruma_api! {
     response: {
         /// The specific push rule.
         #[ruma_api(body)]
-        pub rule: PushRule
+        pub rule: AnyPushRule
```

Comment:
Considering this PR touches this anyway, can you add a trailing comma here? Just noticed it was missing.